File: Code/贺含悦/VJudge/week1/J.java
/*************************************************************************
	> File Name: J.java
	> Author: 
	> Mail: 
	> Created Time: 2018年07月24日 星期二 02时06分40秒
 ************************************************************************/
import java.util.*;
public class J {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        while(n-- > 0) {
            int a = sc.nextInt();
            int b = sc.nextInt();
            if(a == zhenYue(b) && b == zhenYue(a)) {
                System.out.println("YES");
            } else {
                System.out.println("NO");
            }
        }
    }
    public static int zhenYue(int n) {
        int sum = 0;
        for(int i = 1; i <= n/2; i++) {
            if(n % i == 0) {
                sum += i;
            }
        }
        return sum;
    }
}

File: Code/田正东/田正东第1周/j.c
#include <stdio.h>
int main()
{
    int a,b,n,i,sum;
	scanf("%d",&n);
	while(n--)
	{   
		sum=0;
		scanf("%d%d",&a,&b);
	    for(i=1;i<a;i++)
		{
		    if(a%i==0)
			{
			    sum+=i;
			}
		}
		if(sum==b)
			printf("YES\n");
		else
			printf("NO\n");
	}
	return 0;
}
